app/views/admin/loans/timeline/table/_timeline.html.slim

```diff
@@ -0,0 +1,11 @@
+.timeline-table
+  - if @loan.root_timeline_entry
+    - if @loan.root_timeline_entry.is_a?(ProjectGroup)
+      table.table
+        = render "admin/loans/timeline/table/group", entry: @loan.root_timeline_entry, is_first: true,
+          max_depth: @loan.root_timeline_entry.max_descendant_depth
+    / If user has created a step through the timeline list, the timeline table is incompatible
```

Comment:
This is a temporary fix. If a loan has project steps, an empty root group should exist. If a new step is created, the step currently becomes the root, which is not what we want. Please see earlier note in pull request.